scripting/pugsetup/natives.sp

```diff
@@ -397,7 +404,7 @@ public int Native_HasPermissions(Handle plugin, int numParams) {
     return isAdmin;
   else if (p == Permission_Leader)
     return isLeader || isAdmin;
-  else if (p == Permission_Captains)
+  else if (p == Permission_Captains && UsingCaptains())
```

Comment:
This change seems potentially problematic. If a permission has `Permission_Captains` using this native will throw an error now.